charts/lakefs/values.yaml

```diff
@@ -109,6 +109,22 @@ extraEnvVars:
 #   timeoutSeconds: 1
 #   initialDelaySeconds: 5
 
+mds:
```

Comment:
Add `config` as well even commented out if there are nothing to set, as a reference for users.